extension/background.js

```diff
@@ -16,39 +16,39 @@ chrome.runtime.onInstalled.addListener(function() {
+    // Redirect user to a demo page, and auto-start the app there.
+	let demoUrl = 'https://en.wikipedia.org/wiki/Reading';
 	chrome.tabs.create({
 	    url: demoUrl,
 	    active: true
 	}, function(tab) {
-		chrome.tabs.sendMessage(tab.id, {command: "toggleUI"}, function(response) {});
+        // Have to wait for new tab to load; see: https://stackoverflow.com/questions/5816373/chrome-tab-create-and-tab-getselected
+        /* TODO: Change this from this obv super hacky strategy. 
+        Problem is: Only want to sendMessage once, AFTER the new tab is loaded. If do nothing, it sends the message to old page, 
+        which isn't very helpful. How to see if on new page? "tab" has some information for this, but not enough to be useful. 
+        It can show you the:            (from https://developer.chrome.com/extensions/tabs#type-Tab)
+        - tab.status ("unloaded", "loading", "completed"), but it never changes from loading to completed, even after it has completed. 
+            Presumably this is bc it just gives us the tab variable instantiated to "loading" once, and it does not update when the page is completed, 
+            which makes sense. So we cannot have our sendMessage condition based on this variable. 
+        - tab.url will give an empty string, since the tab has not yet committed. 
+        - tab.pendingUrl is always equal to the demoUrl, so we cannot use it to see if we are succesfully at that tab yet. 
+        The stack overflow recommends using onUpdated, but this results in a listener that gets called for every tab update. 
+        You have to build a condition in there, except we cannot look for the URL, bc then it would happen every time we visit that url. 
+        And so on. This works for now, though it incurs an unwanted delay:
+        */
+        setTimeout(function() {
+                chrome.tabs.sendMessage(tab.id, {command: "startTutorial"}, function(response) {
+                    chrome.tabs.sendMessage(tab.id, {command: "toggleUITutorial"}, function(response) {})
+                });
+        }, 3000);      
+    
 	});
+});
 
-	// Creates a new window tutorial window.
-	// TODO: Fill welcome_popup.html with pretty content. 
-	chrome.tabs.create({
-        url: chrome.extension.getURL('welcome_popup.html'),
-        active: false
-    }, function(tab) {
-        // After the tab has been created, open a window to inject the tab
-        // For options documentation, please see
-        // https://developer.chrome.com/extensions/windows#method-create
-        chrome.windows.create({
-            tabId: tab.id,
-            type: 'popup',
-            focused: true,
-            left: 1000,
-            top: 100,
-            width: 400,
-            height: 300,
-            type: "popup"
-        });
-    });
-});
\ No newline at end of file
+// Take user to survey after uninstall
+chrome.runtime.setUninstallURL("https://www.readerease.com/uninstall");
```

Comment:
Just need to make this on website, but can be handled after-publish pre-approval.

Reply:
Very nice!